cd/application-deployment/dev/dev.env

```diff
@@ -0,0 +1,57 @@
+ACCEPT_RECIPIENT_IDENTIFIERS_ENABLED=True
+API_HOST_NAME=https://dev.api.notifications.va.gov
+API_MESSAGE_LIMIT_ENABLED=True
+API_RATE_LIMIT_ENABLED=True
+ATTACHMENTS_BUCKET=dev-notifications-va-gov-attachments
+AWS_PINPOINT_APP_ID=df55c01206b742d2946ef226410af94f
+AWS_SES_EMAIL_FROM_USER=dev-do-not-reply
+CHECK_GITHUB_SCOPE_ENABLED=True
+CHECK_TEMPLATE_NAME_EXISTS_ENABLED=True
+COMP_AND_PEN_DYNAMODB_NAME=dev-bip-payment-notification-table
+COMP_AND_PEN_MESSAGES_ENABLED=True
+DD_ENV=dev
+DD_PROFILING_ENABLED=True
+DD_PROFILING_ENABLE_CODE_PROVENANCE=True
+DD_SERVICE=celery-beat
+DD_SITE=ddog-gov.com
+EMAIL_ATTACHMENTS_ENABLED=True
+EMAIL_PASSWORD_LOGIN_ENABLED=True
+EMAIL_PROVIDER_SELECTION_STRATEGY_LABEL=LOAD_BALANCING
+FLASK_APP=run_celery_beat.py
+GA4_URL=https://www.google-analytics.com/mp/collect
+GITHUB_LOGIN_ENABLED=True
+GOOGLE_ANALYTICS_ENABLED=True
+GRANICUS_URL=https://stage-tms.govdelivery.com
+MPI_URL=https://int.services.eauth.va.gov:9303/int
+NIGHTLY_NOTIF_CSV_ENABLED=True
+NOTIFICATION_FAILURE_REASON_ENABLED=True
+NOTIFICATION_QUEUE_PREFIX=dev-notification-
+NOTIFY_EMAIL_FROM_USER=stage-notifications
+NOTIFY_ENVIRONMENT=development
+PINPOINT_INBOUND_SMS_ENABLED=True
+PINPOINT_RECEIPTS_ENABLED=True
+PLATFORM_STATS_ENABLED=True
+PROVIDER_STRATEGIES_ENABLED=True
+PUSH_NOTIFICATIONS_ENABLED=True
+REDIS_ENABLED=True
+SESSION_COOKIE_SECURE=True
+SMS_PROVIDER_SELECTION_STRATEGY_LABEL=HIGHEST_PRIORITY
+SMS_SENDER_RATE_LIMIT_ENABLED=True
+STATSD_HOST=localhost
+TEMPLATE_SERVICE_PROVIDERS_ENABLED=True
+TWILIO_ACCOUNT_SID=fake
+TWILIO_AUTH_TOKEN=fake
+UI_HOST_NAME=https://dev.notifications.va.gov
+VANOTIFY_SSL_CERT_PATH=/app/certs/vanotify_ssl_cert.pem
+VANOTIFY_SSL_KEY_PATH=/app/certs/vanotify_ssl_key.pem
+VA_FLAGSHIP_APP_SID=A20623E2321D4053A6C34C9307C6C221
+VA_ONSITE_URL=https://staging-api.va.gov
+VA_PROFILE_EMAIL_STATUS_ENABLED=True
+VA_PROFILE_URL=https://int.vaprofile.va.gov
+VA_PROFILE_V3_COMBINE_CONTACT_INFO_AND_PERMISSIONS_LOOKUP=True
+VA_PROFILE_V3_IDENTIFY_MOBILE_TELEPHONE_NUMBERS=True
```

Comment:
we just removed these feature flags yesterday, please remove

Reply:
Good catch! I'll remove those real quick